src/ProxyManager/ProxyGenerator/Util/PublicScopeSimulator.php

```diff
@@ -96,23 +104,29 @@ public static function getPublicAccessSimulationCode(
      *
      * @return string
      */
-    private static function getUndefinedPropertyNotice(string $operationType, string $nameParameter) : string
+    private static function getUndefinedPropertyNotice(string $operationType, string $nameParameter, string $interfaceName = null) : string
     {
         if (static::OPERATION_GET !== $operationType) {
             return '';
         }
 
-        return '    $backtrace = debug_backtrace(false);' . "\n"
+        $code = '    $backtrace = debug_backtrace(false, 1);' . "\n"
             . '    trigger_error(' . "\n"
             . '        sprintf(' . "\n"
             . '            \'Undefined property: %s::$%s in %s on line %s\',' . "\n"
-            . '            get_parent_class($this),' . "\n"
+            . '            ' . (null !== $interfaceName ? var_export($interfaceName, true) : 'get_parent_class($this)') . ',' . "\n"
```

Comment:
Would it make sense to pass in the parent symbol name, instead of relying on `get_parent_class()` at runtime at all?